server_addon/houdini/client/ayon_houdini/plugins/publish/validate_usd_render_arnold.py

```diff
@@ -35,7 +35,7 @@ def process(self, instance):
         )
         render_chunk_size = submission_data.get("chunk", 1)
         export_chunk_size = submission_data.get("export_chunk", 1)
-        usd_file_per_frame = "$F" in instance.data["ifdFile"]
+        usd_file_per_frame = instance.data["farm"]
```

Comment:
This change is incorrect, sorry. We render our USD render `usd` files as singular files, one big chunk instead of per frame - even on the farm because the files are smaller relatively (no duplication of the same data structure) and the server has better "in-memory" load of the one file and the USD files from render scene is usually small due to containing 99% references/sublayers instead of local geometry.

Reply:
Oh, I miss interpreted the meaning of that line! 
So, in the current implementation `"ifdFile"` exists only if the render target was set to `"farm_split"` aka `"Farm Rendering - Split export & render jobs"`. 
So, this validator will crash for other render targets. because of a key error.